docusaurus/docs/Android/04-compose/05-message-components/04-message-composer.mdx

```diff
@@ -135,11 +137,11 @@ fun MessageComposer(
 )
 ```
 
-* `modifier`: Modifier for root component styling. This is useful to set the composer alignment, size, padding and more.
-* `headerContent`: Slot that's shown as the header. By default `MessageInputOptions` that show when a message is being edited, or we're replying to another message.
+* `modifier`: Modifier for root component styling. Used for setting the composer alignment, size, padding and more.
+* `headerContent`: Slot that's shown as the header. By default hosts `MessageInputOptions` which show when a message is being edited, or we're replying to another message.
 * `footerContent`: Slot that's shown as the footer. Present in the Thread mode and allows the user to also send messages from threads to channels.
-* `mentionPopupContent`: Slot that acts as a popup. By default it shows the `MentionSuggestionList` when the user starts the mention action.
```

Comment:
`MentionSuggestionList` is a name, therefor I don't think using the definite article is grammatically correct.
It's tempting to do so because we read it as 3 separate words, but in reality it's a name.